conda_env/cli/main_create.py

```diff
@@ -41,13 +46,17 @@ def configure_parser(sub_parsers):
         default=None,
     )
 
+
     p.add_argument(
         '-f', '--file',
-        action='store',
         help='environment definition (default: environment.yml)',
         default='environment.yml',
     )
     p.add_argument(
+        '--binstar',
```

Comment:
Using a `--binstar` isn't the correct way to do this.  As outlined in #27, the idea is to allow multiple lookups.  I'm using [hub][] as the inspiration.  The UI would then be:

    conda create env tswicegood/stats
    conda create env sean/geo

Both of these are probably outside the scope of this PR though, so I'd say keep that all in a PR related to #27.

[hub]: https://github.com/github/hub/